batch7crud-roll10/src/main/java/com/lftechnology/batch7crud/dao/StudentDao.java

```diff
@@ -0,0 +1,188 @@
+package com.lftechnology.batch7crud.dao;
+
+import com.lftechnology.batch7crud.entity.Student;
+import com.lftechnology.batch7crud.exception.DataException;
+import com.lftechnology.batch7crud.utils.DbConnection;
+
+import javax.xml.crypto.Data;
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * @Author binodnme
+ * Created on 1/14/16
+ */
+public class StudentDao {
+    private static Logger logger = Logger.getLogger(StudentDao.class.getName());
+
+    public List<Student> fetch(Integer offset, Integer limit) throws DataException {
+        List<Student> studentList = new ArrayList<Student>();
+        try {
+            Connection conn = DbConnection.getConnection();
+
+            String query = "select * from student limit " + limit + " Offset " + offset;
+            Statement stmt = conn.createStatement();
+            ResultSet studentResult = stmt.executeQuery(query);
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) "offset" is provided externally to the method and not sanitized before use. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2077)